src/Stats.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+namespace reactivesocket {
+// forward decl for NoopStats
+class NoopStats;
+
+class Stats {
+ public:
+  static NoopStats noop;
```

Comment:
The diff changes dropped the comments, but my last comment was "What does that look like?"